### ISSUE TITLE
Sample in the readme should need to update

### ISSUE THREAD

**Issue by rajithacharith** [asgardeo/asgardeo-auth-node-sdk#24]
**Describe the issue:**
Sample in the readme should need to update.
```
const { AsgardeoAuth } = require('@asgardeo/auth-nodejs-sdk');
const authClient = new AsgardeoAuth(config);
````
needs to be updated to
```
const { AsgardeoNodeClient } = require("@asgardeo/auth-node");
const authClient = new AsgardeoNodeClient(config);
```

**Comment by Suvink** [comment]
Fixed with #23

**Comment by rajithacharith** [comment]
Closing the issue since fixed with https://github.com/asgardeo/asgardeo-auth-node-sdk/pull/23
